### ISSUE TITLE
Result of validate_address_usps is NULL

### ISSUE THREAD

**Issue by PublicRadioCorgi** [hansthompson/rusps#7]
Thanks for updating this package - it's exciting that you're returning to it. Unfortunately, I am consistantly getting back a NULL result set when I use the function - I have properly set my username from usps, but can't get back a result. I'm unable to see any detail behind the 'NULL' as I can't view the actual API response from USPS (or I don't know how to anyway within R).

**Comment by PublicRadioCorgi** [comment]
I've tried creating my own version of the validate address function and found that the usps API is returning the error below 

`$Number
[1] "80040B1A"

$Description
[1] "Authorization failure.  Perhaps username and/or password is incorrect."

$Source
[1] "USPSCOM::DoAuth"
`

From what I can see in the package, there is no way to define my password for use in the function. Is that correct? It seems USPS requires the password they provided me to use the API?

**Comment by PublicRadioCorgi** [comment]
I think I figured the error out - your function looks for arguments of  function(street, city, state, username), but the example you provided to run the function uses  function( username, street, city, state). i.e. username listed first in list of args rather than last. I changed my function arg order and it's working beautifully. 

Might want to just update your 'readme' file to reflect this. :)
